### PR TITLE
* [e2e] Fix daily testing errors and fails

### PR DESCRIPTION
**Pull request by lanfon72**
# Changes
Based on `harvester-install-and-test-e2e-daily` build `#510` and `#511`

## Skipped: old `get_vlan` API has been removed.
- [x] **Error** scenarios/test_vm_networking.py::TestVMNetworking::test_ssh_to_vm::setup
- [x] **Error** scenarios/test_vm_networking.py::TestVMNetworking::test_vlan_ip_pingable_after_reboot::setup
- [x] **Error** scenarios/test_vm_networking.py::TestVMNetworking::test_add_vlan_network_to_vm::setup
- [x] **Error** scenarios/test_vm_networking.py::TestVMNetworking::test_remove_vlan_network_from_vm::setup
- [x] **Error** scenarios/test_vm_networking.py::test_two_vms_on_same_vlan::setup
- [x] **Error** scenarios/test_vm_networking.py::test_update_vm_management_network_to_vlan::setup
- [x] **Error** scenarios/test_vm_networking.py::test_update_vm_vlan_network_to_management::setup
- [x] **Error** scenarios/test_vm_networking.py::test_vm_network_with_bogus_vlan::setup
- [x] **Error** scenarios/test_vm_networking.py::test_create_update_vm_user_data::setup
- [x] **Error** scenarios/test_vm_networking.py::test_create_vm_using_terraform::setup
- [x] **Error** integration/test_terraform.py::test_create_network_using_terraform::setup

## Fixture naming conflict
- [x] **Error**  integration/test_vm_functions.py::test_minimal_vm::setup
- [x] **Error**  integration/test_vm_functions.py::test_create_stopped_vm::setup
- [x] **Error**  integration/test_vm_functions.py::TestVMClone::test_clone_running_vm::setup
- [x] **Error**  integration/test_vm_functions.py::TestVMClone::test_clone_stopped_vm::setup
- [x] **Error**  integration/test_vm_functions.py::TestVMWithVolumes::test_create_with_two_volumes::setup
- [x] **Error**  integration/test_vm_functions.py::TestVMWithVolumes::test_create_with_existing_volume::setup
- [x] **Failed** integration/test_images.py::test_create_with_url
- [x] **Failed** integration/test_volumes.py::test_create_volume_backing_image

## Others: (will not fix in this PR, might need more test or investigate)
- [ ] **Error**   integration/test_upgrade.py::TestThreeNodesUpgrade::test_perform_upgrade::setup
    know issue, it's template code for upgrade
- [ ] **Failed**  integration/test_hosts.py::test_maintenance_mode_trigger_vm_migrate
- [ ] **Failed**  integration/test_rancher_integration.py::TestRKE::test_create_rke1
- [ ] **Failed**  apis/test_hosts.py::test_maintenance_mode
- [ ] **Failed**  apis/test_images.py::test_create_with_invalid_url
- [ ] **Failed**  apis/test_networks.py::TestNetworksNegative::test_create_with_invalid_id[0]
- [ ] **Failed**  apis/test_networks.py::TestNetworksNegative::test_create_with_invalid_id[4095]
- [ ] **Failed**  apis/test_networks.py::TestNetworks::test_create
- [ ] **Failed**  scenarios/test_create_vm.py::test_create_vm_do_not_start
- [ ] **Failed**  scenarios/test_create_vm.py::test_create_vm_on_available_cpu_node
- [ ] **Failed**  scenarios/test_create_vm.py::test_update_vm_on_available_cpu_node
- [ ] **Failed**  scenarios/test_create_vm.py::test_host_maintenance_mode

### CODE REVIEW

File: harvester_e2e_tests/scenarios/test_vm_networking.py [hunk 1]
```diff
@@ -299,6 +299,7 @@ def backup_restore_chained_backups(request, admin_session,
                                            backuptarget, backup_json)
 
 
+@pytest.mark.skip("get_vlan API removed")
```
Comment (khushboo-rancher) [inline harvester_e2e_tests/scenarios/test_vm_networking.py:302]: Going forward, we need to completely remove the legacy code.

**Review by khushboo-rancher — COMMENTED** [review]
LGTM in general,
Do we want to skip the tests and see the results first before we remove the legacy code, is that why these are skipped not removed for now?

**Comment by lanfon72** [comment]
> LGTM in general,
> 
> Do we want to skip the tests and see the results first before we remove the legacy code, is that why these are skipped not removed for now?

My thought is, remove all legacy code files when all test cases been marked `skip`, so that we can just tag one commit for legacy code.

**Comment by khushboo-rancher** [comment]
>My thought is, remove all legacy code files when all test cases been marked skip, so that we can just tag one commit for legacy code.

Sounds good!

**Review by khushboo-rancher — APPROVED** [review]

**Pull request merged**